docs/usage.md

```diff
@@ -225,13 +225,14 @@ Frames that match these regular expressions will be not considered during analys
 
 ## casr-cli
 
-App provides text-based user interface to view CASR reports
+App provides text-based user interface to view CASR reports and print common statistic all
```

Comment:
and print joint statistics for multiple reports

Reply:
maybe for all? Multiple != all

Reply:
> maybe for all? Multiple != all

ok

Reply:
done